fetch.sh

```diff
@@ -183,6 +185,24 @@ getwindowmanager () {
     fi
 }
 
+
+# Font (Configurable with "-F" and "--font" at launch)
```

Comment:
I don't know if I want to add this as it only works with terminals that use Xresources/Xdefaults. We might be able to support more terminals by detecting terminal emulator and then checking default config locations for fonts.

I'll work on it.